services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationEntity.kt

```diff
@@ -35,9 +35,9 @@ data class ApplicationEntity(
             applicationIdentifier,
             applicationType,
             applicationData,
-            hanke.hankeTunnus!!,
+            hanke.hankeTunnus,
         )
 
     /** An application must belong to a Hanke. Thus, hankeTunnus must be present. */
-    fun hankeTunnus(): String = hanke.hankeTunnus!!
+    fun hankeTunnus(): String = hanke.hankeTunnus
```

Comment:
I'm not sure if this is actually _needed_ anymore, but I guess it may be a convenient way to get the tunnus. Feel free to ignore.